src/frontend/planner_test/tests/testdata/output/with_ordinality.yaml

```diff
@@ -151,40 +151,25 @@
         │       └─BatchScan { table: t, columns: [t.arr], distribution: SomeShard }
         └─BatchProjectSet { select_list: [$0, Unnest($0)] }
           └─BatchHashAgg { group_key: [t.arr], aggs: [] }
-            └─BatchHashJoin { type: Inner, predicate: t.arr IS NOT DISTINCT FROM t.arr, output: [t.arr] }
-              ├─BatchExchange { order: [], dist: HashShard(t.arr) }
-              │ └─BatchScan { table: t, columns: [t.arr], distribution: SomeShard }
-              └─BatchProjectSet { select_list: [$0, Unnest($0)] }
-                └─BatchHashAgg { group_key: [t.arr], aggs: [] }
-                  └─BatchExchange { order: [], dist: HashShard(t.arr) }
-                    └─BatchScan { table: t, columns: [t.arr], distribution: SomeShard }
+            └─BatchExchange { order: [], dist: HashShard(t.arr) }
+              └─BatchScan { table: t, columns: [t.arr], distribution: SomeShard }
```

Comment:
Because the initial logical plan of this case is:
```sql
 LogicalProject { exprs: [t.x, t.arr, unnest, ordinality, unnest, ordinality] }
 └─LogicalApply { type: Inner, on: true, correlated_id: 1 }
   ├─LogicalApply { type: Inner, on: true, correlated_id: 2 }
   │ ├─LogicalScan { table: t, columns: [x, arr, _row_id] }
   │ └─LogicalTableFunction { table_function: Unnest(CorrelatedInputRef { index: 1, correlated_id: 2 }) }
   └─LogicalTableFunction { table_function: Unnest(CorrelatedInputRef { index: 1, correlated_id: 1 }) }
```
It is the same shape as the multi scalar subqueries.